extra_data/components.py

```diff
@@ -176,6 +177,31 @@ def __init__(self, data: DataCollection, detector_name=None, modules=None,
     def __getitem__(self, item):
         return MultimodKeyData(self, item)
 
+    def masked_data(self, key=None, *, mask_bits=None, masked_value=np.nan):
```

Comment:
Since bit logic is not generally intuitive for pragmatic programmers, allowing to pass bad pixel bits by iterable may be nice here as well.

Reply:
That makes sense. I guess we should also expose the bad pixel flags somewhere - I don't think we have those at the moment. Maybe they belong in `extra.calibration` ?

Reply:
Definitely, EXtra was the intended target for some time now. `extra.calibration` makes sense given it's used in calibration data.

Reply:
Just to record here, I opened https://github.com/European-XFEL/EXtra/pull/172 to do that.